Migrations error out in the CI test

Below is the interesting part of [this](https://github.com/OpenImaging/miqa/runs/4445113581?check_suite_focus=true).
```text
...
ERROR: InvocationError for command /home/runner/work/miqa/miqa/.tox/check-migrations/bin/python ./manage.py makemigrations --check --dry-run (exited with code 1)
___________________________________ summary ____________________________________
  lint: commands succeeded
  test: commands succeeded
ERROR:   check-migrations: commands failed
Error: Process completed with exit code 1.
```

--- a/setup.py
+++ b/setup.py
@@ -37,7 +37,7 @@ setup(
     include_package_data=True,
     install_requires=[
         'celery',
-        'django>=3.2',
+        'django>=3.2,<4.0',
         'django-allauth',
         'django-configurations[database,email]',
         'django-extensions',

